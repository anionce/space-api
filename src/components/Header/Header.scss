@import '../../assets/styles/variables';

.header {
	text-align: center;
	margin: 16px;

	a {
		color: $color-primary;
	}

	.header-logo {
		height: 150px;
	}
}
