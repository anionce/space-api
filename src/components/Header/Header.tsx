import { Link } from 'react-router-dom';
import './Header.scss';
import spaceLogo from '../../assets/img/spaceLogo.svg';
import React from 'react';

export const Header = () => {
	return (
		<div className='header'>
			<Link to='/'>
				<img className='header-logo' alt='header-logo' src={spaceLogo}></img>
			</Link>
		</div>
	);
};
