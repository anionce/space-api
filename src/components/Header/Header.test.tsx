import React from 'react';
import { screen } from '@testing-library/react';
import { customRender } from '../../assets/tests/utils';
import { Header } from './Header';

test('renders correcly', () => {
	const { container } = customRender(<Header />);

	expect(container).toMatchSnapshot();
});

test('renders elements correctly', () => {
	customRender(<Header />);

	expect(screen.getByAltText(/header-logo/i)).toBeInTheDocument();
});
