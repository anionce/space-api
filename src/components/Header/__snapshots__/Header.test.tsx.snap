// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correcly 1`] = `
<div>
  <div
    class="header"
  >
    <a
      href="/"
    >
      <img
        alt="header-logo"
        class="header-logo"
        src="spaceLogo.svg"
      />
    </a>
  </div>
</div>
`;
